CodeEditModules/Modules/Git/src/Client/Live.swift

```diff
@@ -59,6 +59,25 @@ public extension GitClient {
             }
         }
 
+        /// Displays paths that have differences between the index file and the current HEAD commit,
```

Comment:
I would put this comment above the closure in the interface

Reply:
marked as unresolved since it wasn't changed

Reply:
Not enough information to do what you are asking for

Reply:
marked as unresolved since it wasn't changed

Reply:
What I'm suggesting to do here is moving this comment above the closure declaration which you can find in the Interface

Reply:
What interface... Direct me to the line

Reply:
https://github.com/CodeEditApp/CodeEdit/blob/c9ffbeed29e16decba3e4b4e00c75bb7b781f04e/CodeEditModules/Modules/Git/src/Client/Interface.swift#L16